grafana/overlays/moc/smaug/blackbox-exporter/service-monitor.yaml

```diff
@@ -21,6 +21,76 @@ spec:
           - 'https://jupyterhub-opf-jupyterhub.apps.smaug.na.operate-first.cloud/'
       path: /probe
       targetPort: 9115
+    - interval: 30s
+      metricRelabelings:
+        - sourceLabels:
+            - __address__
+          targetLabel: __param_target
+        - sourceLabels:
+            - __param_target
+          targetLabel: instance
+        - replacement: 'https://www.operate-first.cloud/'
+          targetLabel: target
+      params:
+        module:
+          - http_2xx
+        target:
+          - 'https://www.operate-first.cloud/'
+      path: /probe
+      targetPort: 9115
+    - interval: 30s
+      metricRelabelings:
+        - sourceLabels:
+            - __address__
+          targetLabel: __param_target
+        - sourceLabels:
+            - __param_target
+          targetLabel: instance
+        - replacement: 'https://thanos-query-frontend-opf-observatorium.apps.smaug.na.operate-first.cloud/'
+          targetLabel: target
+      params:
+        module:
+          - http_2xx
+        target:
+          - 'https://thanos-query-frontend-opf-observatorium.apps.smaug.na.operate-first.cloud/'
+      path: /probe
+      targetPort: 9115
+    - interval: 30s
+      metricRelabelings:
+        - sourceLabels:
+            - __address__
+          targetLabel: __param_target
+        - sourceLabels:
+            - __param_target
+          targetLabel: instance
+        - replacement: 'https://trino.operate-first.cloud/'
+          targetLabel: target
+      params:
+        module:
+          - http_2xx
+        target:
+          - 'https://trino.operate-first.cloud/'
+      path: /probe
+      targetPort: 9115
+    - interval: 30s
+      metricRelabelings:
+        - sourceLabels:
+            - __address__
+          targetLabel: __param_target
+        - sourceLabels:
+            - __param_target
+          targetLabel: instance
+        - replacement: 'http://cloudbeaver-opf-trino.apps.smaug.na.operate-first.cloud/'
+          targetLabel: target
+      params:
+        module:
+          - http_2xx
+        target:
+          - 'http://cloudbeaver-opf-trino.apps.smaug.na.operate-first.cloud/'
+      path: /probe
+      targetPort: 9115
   namespaceSelector:
+    spec:
+      selector: {}
```

Comment:
should be in the root spec ([example](https://github.com/operate-first/apps/blob/master/kfdefs/base/jupyterhub/servicemonitor.yaml#L12))

Reply:
done